nebula_decoders/include/nebula_decoders/nebula_decoders_hesai/decoders/hesai_decoder.hpp

```diff
@@ -41,84 +40,73 @@ namespace nebula::drivers
 
 struct HesaiDecodeFilteredInfo
 {
-  uint16_t distance_filtered_count = 0;
-  uint16_t fov_filtered_count = 0;
-  uint16_t timestamp_filtered_count = 0;
-  uint16_t invalid_point_count = 0;
-  uint16_t multiple_return_point_count = 0;
-  uint16_t mutliple_return_point_count = 0;
-  uint16_t total_kept_point_count = 0;
-  uint16_t invalid_packet_count = 0;
-  float cloud_distance_min_m = 0;
-  float cloud_distance_max_m = 0;
-  float cloud_azimuth_min_rad = 0;
-  float cloud_azimuth_max_rad = 0;
-  uint64_t packet_timestamp_min_ns = 0;
-  uint64_t packet_timestamp_max_ns = 0;
-
-  void clear()
-  {
-    distance_filtered_count = 0;
-    fov_filtered_count = 0;
-    timestamp_filtered_count = 0;
-    invalid_point_count = 0;
-    multiple_return_point_count = 0;
-    mutliple_return_point_count = 0;
-    total_kept_point_count = 0;
-    invalid_packet_count = 0;
-    cloud_distance_min_m = 0;
-    cloud_distance_max_m = 0;
-    cloud_azimuth_min_rad = 0;
-    cloud_azimuth_max_rad = 0;
-    packet_timestamp_min_ns = 0;
-    packet_timestamp_max_ns = 0;
-  }
+  uint64_t distance_filtered_count = 0;
+  uint64_t fov_filtered_count = 0;
+  uint64_t invalid_point_count = 0;
+  uint64_t identical_filtered_count = 0;
+  uint64_t multiple_return_filtered_count = 0;
+  uint64_t total_kept_point_count = 0;
+  uint64_t invalid_packet_count = 0;
+  float cloud_distance_min_m = std::numeric_limits<float>::infinity();
+  float cloud_distance_max_m = std::numeric_limits<float>::lowest();
```

Comment:
While also okay, `lowest()` represents the lowest _finite_ number, so for consistency, and to signal that the initial value is not a valid one, let's change this to `-infinity()` instead:
```suggestion
  float cloud_distance_max_m = -std::numeric_limits<float>::infinity();
```